apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.GenerateClient/Program.cs

```diff
@@ -33,12 +33,48 @@ class Program
     {
         private const string AppProfileIdFieldName = "_appProfileId";
         private const string AppProfileIdPropertyName = "AppProfileId";
-        private const string GetClientMethodName = "GetClient";
+        private const string GetUnderlyingClientMethodName = "GetUnderlyingClient";
```

Comment:
It does currently return a field, but that's just a stub for the real implementation (being worked on in parallel), which will cycle through multiple underlying clients. This method, just renamed because it is now protected, could (and probably will) return a different client on each invocation.

Reply:
Ah yes - sorry for forgetting about that. A method makes a lot of sense then.